code/api/package.json

```diff
@@ -9,6 +9,7 @@
     "setup:db": "npm run db:migrate && npm run db:seed",
     "db:migrate": "./node_modules/.bin/sequelize db:migrate",
     "db:seed": "./node_modules/.bin/sequelize db:seed:all",
+    "db:undo_seed": "./node_modules/.bin/sequelize db:seed:undo:all",
```

Comment:
Added a script to undo the seeds so that we could update the user seed file with the style attribute and reseed.